catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/pojo/FieldFilter.java

```diff
@@ -35,10 +38,26 @@
   @SuppressWarnings("squid:S1068" /* Needed for serialization */)
   private final Set<String> descriptors;
 
-  public FieldFilter(Metacard metacard, Set<String> descriptors) {
+  @JsonProperty("creator")
+  private String creator;
```

Comment:
❓ Any reason for storing these properties in this `FieldFilter` as opposed to the `CommonTemplate`?